src/Features/CSharpTest/Snippets/CSharpPropSnippetProviderTests.cs

```diff
@@ -65,4 +66,8 @@ interface MyInterface
             }
             """);
     }
+
+    [Theory]
+    [MemberData(nameof(CommonSnippetTestData.AllAccessibilityModifiers), MemberType = typeof(CommonSnippetTestData))]
+    public override Task InsertSnippetAfterAllowedAccessibilityModifierTest(string modifier) => base.InsertSnippetAfterAllowedAccessibilityModifierTest(modifier);
```

Comment:
Please add a line break like that for all these tests. This is _too_ long line. It was just not that obvious from side-by-side diff
```suggestion
    public override Task InsertSnippetAfterAllowedAccessibilityModifierTest(string modifier)
        => base.InsertSnippetAfterAllowedAccessibilityModifierTest(modifier);
```

Reply:
fixed